### PR TITLE
includes information about the RSA256 algorithm we now support

### PR DESCRIPTION
**Pull request by PippaClarkGDS**
## Why

We now support RSA256 as an alternative algorithm.

## What

Includes a line about RSA256 and when users would support this. 

## Technical writer support

Pippa and James paired.

### CODE REVIEW

File: source/integrate-with-integration-environment/manage-your-service-s-configuration.html.md.erb [hunk 1]
```diff
@@ -46,6 +46,8 @@ To register your service, you must send:
 * the URL you’d like your users redirected to if they log out of your service - if you do not specify one, your users will be redirected to the [default GOV.UK sign out page](https://signin.account.gov.uk/signed-out)
 * your `sector_identifier_uri` with the identifier for your sector
 
+By default, GOV.UK One Login will sign the `id_token` JSON Web Token (JWT) using the `ES256` algorithm but some third party tooling does not support `ES256`. If your service needs an alternative algorithm, we can sign your JWT `id_token` using the `RSA256` algorithm. Let us know if you need this when you register your service.
```
Comment (NathanD-GDS) [inline source/integrate-with-integration-environment/manage-your-service-s-configuration.html.md.erb:49]: Minor thing, but you reverse this phrase within this paragraph:

> `id_token` JSON Web Token (JWT)

and

> JWT `id_token`

I'd stick with "`id_token` JWT".

Comment (NathanD-GDS) [reply source/integrate-with-integration-environment/manage-your-service-s-configuration.html.md.erb:49]: I don't know if this comment is clear. Let me know if it isn't!

**Comment by NathanD-GDS** [comment]
Looks good to me! Just a really minor suggestion.

File: source/integrate-with-integration-environment/manage-your-service-s-configuration.html.md.erb (hunk 1 above)
Comment (PippaClarkGDS) [reply source/integrate-with-integration-environment/manage-your-service-s-configuration.html.md.erb]: good catch! Updating now

**Comment by sabrinaharris** [comment]
Looks good to me! 🚀

**Review by JHjava — APPROVED** [review]

**Review by sabrinaharris — APPROVED** [review]

**Pull request merged**